luda-editor/luda-editor-client/src/app/types/app_context.rs

```diff
@@ -0,0 +1,9 @@
+use super::{CameraAngleImageLoader, MetaContainer};
+use crate::app::storage::GithubStorage;
+use std::sync::Arc;
+
+pub struct AppContext {
+    pub storage: Arc<dyn GithubStorage>,
+    pub meta_container: Arc<MetaContainer>,
+    pub camera_angle_image_loader: Arc<dyn CameraAngleImageLoader>,
+}
```

Comment:
`camera_angle_image_loader` is exactly same as `namui's image_loader`. But it uses `GithubStorage` instead.